app/database/schema.py

```diff
@@ -0,0 +1,102 @@
+from sqlalchemy import Column, Integer, String, Float, Boolean
+from sqlalchemy.ext.declarative import declarative_base
+
+
+Base = declarative_base()
+
+
+class BuildStateSchema(Base):
+    """Describes the state of a Build"""
+    __tablename__ = 'builds'
+
+    build_id = Column(Integer, primary_key=True, autoincrement=True)
+    completed = Column(Boolean)
+
+
+class BuildMetaSchema(Base):
```

Comment:
I originally split them apart when I was breaking down all the attributes of a build -- I'm sure there's tons of room for improvement on how these tables are structured (for example, condensing tables like these into one). Originally it was just a little more clear where the values were coming from and where they belong